docs/design.ltx

```diff
@@ -0,0 +1,357 @@
+
+When testing the progress of the training on the neural network algorithm, the algorithm will be run on images with known land surfaces that were not used in the training stages. If the algorithm correctly classifies at least 80\% of the pixels in at least 80\% of the images used for testing, the training will be considered sufficient. Otherwise, more samples will be required in the training and the algorithm will need to be tested again. In general, small sample sizes do not produce reliable results when making predictions, so making sure the training has used enough AVIRIS images is important. 
+
+After the training stage, AVIRIS images with unknown land surfaces will be processed with the trained algorithm. The algorithm will classify each pixel into a class representing a land surface and that data will be used for further analysis (see design items \ref{subsec:identifymining}, \ref{subsec:preprocessing}, and \ref{subsec:correlate}) and possibly be output into an image to be used on the homepage. 
+
+Something to consider when training and using this algorithm for analysis is spectral separability. Some types of minerals and other land surfaces have very similar relationships between reflectance and wavelength and can be cause the algorithm’s accuracy to suffer if they are separated into different classes \cite{separability}. It is up to the developers to determine if the particular minerals or other land surfaces that are not very separable should be put into the same class or kept separate depending on if the minerals or other land surfaces in question are important enough by themselves in some way to warrant a slightly lower accuracy.
+
+
+% xiaomei: requirement 3.4.2: identify mining
+\subsection{Identify Mining}
+\label{subsec:identifymining}
+
+After we get the mineral identification and classification data, we need to identify the regions of mining activity. The data is further processed and possibly combined with geographic information about geology and mines. The goal of identifying mining is to accurately locate regions in which mining is taking place in a form that is suitable for both human inspection and further processing. This step creates input data for further processing of the project and the data evaluation.
```

Comment:
It's also important to state that because AVIRIS and AVIRIS-NG data is collected as swath (flight lines) as oppose to global coverage, we are physically restricted to only what is available. This also limits the possibility of us combining the data with existing thirst party models or databases which may indicate mining activity.